src/dualpicker/dualpicker.component.ts

```diff
@@ -416,9 +416,15 @@ export class DualPickerComponent extends DatePickerBase implements OnChanges {
     updateMinDate(minDate: moment.Moment) {
         this.cal1.minDate = minDate;
         this.cal2.minDate = minDate;
+
+        this.cal1.initCalendar(this.dateFrom, this.minDate, this.maxDate);
```

Comment:
This is essentially equivelent. any reason you didn't use the default values for min/max?  Just curious.